src/parameters/all_parameters.h

```diff
@@ -255,10 +256,22 @@ class AllParameters
     DissipativeNumericalFlux diss_num_flux_type;
 
     /// Type of correction in Flux Reconstruction
-    enum Flux_Reconstruction {cDG, cSD, cHU, cNegative, cNegative2, cPlus, c10Thousand, cHULumped};
+    enum Flux_Reconstruction {cDG, cSD, cHU, cNegative, cNegative2, cPlus, c10Thousand, cHULumped, user_specified_value};
     /// Store flux reconstruction type
     Flux_Reconstruction flux_reconstruction_type;
 
+    /// User specified flux recontruction correction parameter value
+    double FR_user_specified_correction_parameter_value;
+
+    /// User specified flux recontruction correction parameter value
+    int number_ESFR_parameter_values;
+
+    /// User specified flux recontruction correction parameter value
+    double ESFR_parameter_values_start;
+
+    /// User specified flux recontruction correction parameter value
+    double ESFR_parameter_values_end;
```

Comment:
might be best to move these into a separate file similar to the `time_refinement_study` parameters;  also, it is intentional that you've prepended their names with `ESFR` and not `NSFR` ?

Reply:
I think the naming is intentional - the `c` parameter is developed for FR schemes which are stable in L2 for linear advection, hence "energy stable". I think naming them `NSFR` would be misleading as it would indicate that we know these values are stable for nonlinear problems, which we generally need to test by trial and error.

I agree with the comment about moving test-specific parameters elsewhere, though.

Reply:
I haven't looked at it too deeply, but it's also OK for us to have constants defined values for our tests. Not everything needs to be a user-input parameters. At some point, we will have confidence about this test or will have published something about it. Then this parameter will never change again, so user-defined just won't make sense. We just want a range of values for which we know our test will be robust enough to catch new bugs.

Reply:
Ok sounds good, I'll let @ErwanLa resolve this comment